src/Microsoft.Health.Abstractions/Errors/IErrorMessageService.cs

```diff
@@ -0,0 +1,22 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Abstractions.Errors
+{
+    public interface IErrorMessageService
+    {
+        /// <summary>
+        /// Writes a collection of ErrorMessages to a destination
```

Comment:
nit: if you truly wish to refer explicitly to the type (which you don't have to) then you should use the proper tags. Either:

```csharp
/// Asynchronously writes a collection of <see cref="ErrorMessage" /> objects to a destination.
```

OR

```csharp
/// Asynchronously writes a collection of errror messages to a destination.
```